vitals/ReportGenerator.java

```diff
@@ -0,0 +1,109 @@
+package vitals;
+
+import java.util.Map;
+
+public class ReportGenerator {
+	
+	
+	public static  void getBatteryReport() {
+		StringBuilder sb=new StringBuilder();
+		boolean factorViolation=CheckAlertFactors()&&CheckBrechFactors();
+		sb=(BatteryFactorAccumulator.language=="EN")?GenerateReportEN(factorViolation,sb):GenerateReportDE(factorViolation,sb);
+		BatteryFactorAccumulator.VitalFactorExceedMaxLimit.clear();
+		BatteryFactorAccumulator.VitalFactorExceedMinLimit.clear();
+		BatteryFactorAccumulator.VitalFactorReachHighBreach.clear();
+		BatteryFactorAccumulator.VitalFactorReachLowBreach.clear();
+		System.out.println(sb.toString());		
+	}
+
+	private static StringBuilder GenerateReportDE(boolean factorViolation, StringBuilder sb) {
+		// TODO Auto-generated method stub
+		sb=(factorViolation)?printSuccessDE(sb):printFailureDE(sb);
+		return sb;
+	}
+
+	private static StringBuilder GenerateReportEN(boolean factorViolation, StringBuilder sb) {
+		// TODO Auto-generated method stub
+		sb=(factorViolation)?printSuccessEN(sb):printFailureEN(sb);
+		return sb;
+	}
+
+	private static StringBuilder printFailureEN(StringBuilder sb) {
```

Comment:
print function should hold only responsibilities related to print

Reply:
changed logic please check